doc/Building_targets.md

```diff
@@ -62,7 +62,14 @@ Building for Android requires `openjdk-8-jdk` and `p7zip-full`.
 For both Ubuntu and Debian, install the packages:
 
 ```
-sudo apt-get install openjdk-8-jdk p7zip-full
+sudo apt-get install openjdk-8-jdk p7zip-full ncurses
+```
+
+Note: In case a more up-to-date package manager is used (like Arch), then it may be necessary to link the following files because `ncurses` by default does not provide those anymore but are necessary for `android-ndk-r15c`:
+
+```sh
+sudo ln -sf ./libncursesw.so.6.3 ./libncurses.so.5
+sudo ln -sf ./libncursesw.so.6.3 ./libtinfo.so.5
```

Comment:
I don't know Arch very well but there seems to be a proper package instead?

https://aur.archlinux.org/packages/ncurses5-compat-libs

Reply:
there are 2 problems with that package (i had actually tried it):
1. it is a AUR so its not official
2. this package fails because its signature is not correct and seems to not have got and update to fix this

Reply:
It's... more complicated than that.

There *is* an option to keep building ncurses 6 w/ the ncurses 5 ABI, and the very visible result of that flags, besides the actual internal changes, is that it's built w/ the SOVER set to 5 ;).

TL;DR: If you don't have a `libncursesw.so.5`, your ncurses build does NOT, in fact, support the ncurses 5 ABI (and stuff *will* horribly crash at runtime).

Reply:
> TL;DR: If you don't have a libncursesw.so.5, your ncurses build does NOT, in fact, support the ncurses 5 ABI (and stuff will horribly crash at runtime).

but i dont have it(only symlinked to 6.3) and it compiles and runs fine?

Reply:
That's the magic of ABI mismatches: some stuff will work, some stuff will subtly break, some stuff will subtly do things differently, and some stuff will horribly crash ;).

Reply:
@NiLuJe I don't think you're interpreting it correctly, hence why I relented. It's just for clang to display compilation info in funky ways.

Reply:
I.e., ideally they shoudln't have built Clang with `-DLLVM_ENABLE_TERMINFO=on` or included the ncurses library in the NDK.

Reply:
Unless you think that it'll do things subtly differently *other than terminal output*?

Reply:
I can't exclude the fact that it might horribly crash on *some* use-cases when it tries to poke at the wrong data structure or function in libtinfo, yeah :/.

Reply:
Apparently, the usage seems trivial enough that it *might* be safe, but this is a *terrible* recommendation that may have system-wide side-effects ;).

Reply:
If the symlink is the sanest solution Arch users can find, I'd isolate it to the toolchain itself instead of the system ;).

Reply:
Yes!